.github/workflows/node.js.yml

```
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

```suggestion
      matrix:
        fail-fast: false
        node-version: [10.x, 12.x, 14.x, 15.x]
```

By default, GitHub cancels all in-progress jobs if any `matrix` job fails.
Example: if Node 12 fails, the jobs for 10.x, 14.x and 15.x are cancelled.

You can use `fail-fast: false` so that GitHub continues running all tests, even if a test fails.
Example: Node 12 fails, but tests for 10.x, 14.x and 15.x continue running.
This way you can use the tests as a rough guide of which version of Node is having problems.